### ISSUE TITLE
Binary is greyed out when I try to give it full disk access

### ISSUE THREAD

**Issue by 0dragosh** [YuriyGuts/persistent-touch-id-sudo#2]
The `/usr/local/bin/com.yuriyguts.persistent-touch-id-sudo` binary is greyed out and unselectable when attempting to give it Full Disk Access permissions.
<img width="238" alt="Screenshot 2022-06-15 at 16 09 32" src="https://user-images.githubusercontent.com/39906742/173834974-74a90859-68b0-4547-a412-fcc8d3b6b772.png">

**Comment by YuriyGuts** [comment]
Thanks for reporting the issue! I haven't observed this behavior before while using the tool on multiple systems, but I was able to reproduce it just now. Apparently, macOS no longer allows selecting binaries that use the "reverse domain name" naming pattern (`com.domain.*`), so a simple rename should suffice.

If you'd like to try out the tool again, you can simply delete the old binary and the intermediate build directory:
```shell
sudo rm /usr/local/bin/com.yuriyguts.persistent-touch-id-sudo
rm -rf %CLONED_PROJECT_DIR%/build
```

and repeat the steps in the [Installation](https://github.com/YuriyGuts/persistent-touch-id-sudo#installing) section again.

**Comment by YuriyGuts** [comment]
Also, feel free to reopen the issue if the fix in #3 still doesn't work
